docker/cloudbuild.yaml

```diff
@@ -61,6 +61,17 @@ steps:
     docker push getsentry/sentry:$COMMIT_SHA
     docker tag us.gcr.io/$PROJECT_ID/sentry:$COMMIT_SHA getsentry/sentry:latest
     docker push getsentry/sentry:latest
+- name: 'node:12'
+  secretEnv: ['ZEUS_HOOK_BASE']
+  entrypoint: 'bash'
+  args:
+  - '-e'
+  - '-c'
+  - |
+    # Only push to Zeus from releases/*
+    # [[ ! "$BRANCH_NAME" =~ ^release/ ]] && exit 0
+    npx -p "@zeus-ci/cli" zeus job update --build $BUILD_ID --job 1 --ref $COMMIT_SHA --build-label "$(Build.BuildNumber)" --job-label "OSS Packages" --status passed
+    npx -p "@zeus-ci/cli" zeus upload --build $BUILD_ID --job 1 --type "application/x-pywheel+zip" dist/*.whl
```

Comment:
I think setting the job ID to `1` statically should work, but let's double-check that we can have it unique per-build in Zeus.

Reply:
I think the concept is equivalent to how Travis does so this is essentially a step id inside the build.